### ISSUE TITLE
Clarify aria-required-parent/child message for presentation conflict

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4362]
the following code creates a somewhat confusing remediation message:

```htmkl
    <ul role="tablist">
      <li role="presentation" aria-hidden="false">
        <div role="tab" tabindex="0" aria-selected="true">Tab 1</div>
      </li>
    </ul>
```

```
aria-required-children:
  Fix any of the following:
    Element has children which are not allowed: [role=presentation]
    Element uses aria-busy="true" while showing a loader
```
```
aria-required-parent:
  Fix any of the following:
    Required ARIA parent role not present: tablist
```

Using `[role=presentation]` is just fine here, the issue is with the global attribute. The issue is that the presentation role is ignored because of aria-hidden. We should have an explicit message explaining this in both rules.

**Comment by straker** [comment]
It also should give an `presentation-role-conflict` violation which tells you about the conflict , though the message is a bit confusing as well ("does not have global ARIA attribute"?)

```
"Fix all of the following:
  Element does not have global ARIA attribute"
```

Do we need to tell the user 3 times that the role is ignored?
